addons/website/views/website_templates.xml

```diff
@@ -315,6 +315,14 @@
     </xpath>
 </template>
 
+<template id="header_button" inherit_id="website.layout" name="Header Button" active="False">
+    <xpath expr="//ul[@id='top_menu']" position="inside">
+        <li class="nav-item align-self-md-center ml-1 o_header_menu_button">
+            <a href="/contactus" class="btn btn-primary">Contact us</a>
```

Comment:
FIXME for this feature: clicking on this button in edit mode should not ask to edit the menu